exporter/awskinesisexporter/exporter.go

```diff
@@ -6,7 +6,6 @@ package awskinesisexporter // import "github.com/open-telemetry/opentelemetry-co
 import (
 	"context"
 	"errors"
-
```

Comment:
this is unrelated and will cause linter failure

Reply:
fixed it, thanks!